dataquality/loggers/data_logger/text_classification.py

```diff
@@ -1,12 +1,13 @@
 from collections import defaultdict
 from enum import Enum, unique
-from typing import Any, DefaultDict, Dict, Iterable, List, Optional, Union
+from typing import Any, DefaultDict, Dict, Iterable, List, Optional, Union, cast
 
 import numpy as np
 import pandas as pd
 import vaex
 from vaex.dataframe import DataFrame
 
+import dataquality
```

Comment:
Nit: Might be nicer to import the function directly to match other imports below.
```suggestion
from dataquality import set_labels_for_run
```